src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum;
```

Comment:
```suggestion
```
No need for this variable. In simple functions, you may return simple expressions directly

Don't leave commented lines in your code before pushing solutiong